_articles/numba.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: article
+title: Install Numba  on Python3
+description: >
+  Complete instructions on setting up the Numba library in Python for fast, parralell computing using the NVIDIA CUDA toolkit.
+keywords:
+  - PYTHON
+  - NUMBA
+  - NVIDIA
+  - CUDA
+image: http://support.system76.com/images/system76.png
+hidden: false
+section: software-applications
+
+---
+
+## Install on Pop!_OS
+
+### Prerequisits
+
+These instructions assume that the CUDA toolkit is already set up and working on your installation of Pop_OS.
+The Environmental Variable setup assumes the default Bash shell is used in your environment. Should you be using a different one, please set up the environmental variables using that (eg. for zsh, ~/.zshrc).
```

Comment:
You could mention that adding the variables to `/etc/environment` makes them system-wide and shell independent.

Reply:
Thanks mmstick - good idea. I'll include that.